src/mixins/focusOnDoubleClick.js

```diff
@@ -0,0 +1,48 @@
+let timeoutID;
+
+function waitToTriggerOpenAnimation() {
+  return new Promise(resolve => {
+    timeoutID = setTimeout(resolve);
+  });
+}
+
+async function focusNameInput(cellView) {
+  if (cellView.model !== this.shape) {
+    return;
+  }
+
+  const nameInput = document.querySelector('[name="name"]') || document.querySelector('[name="text"]');
+
+  if (!nameInput || document.activeElement === nameInput) {
+    return;
+  }
+
+  const configurationAccordion = document.getElementById('accordion-button-Configuration');
+  if (configurationAccordion.getAttribute('aria-expanded') === 'false') {
+    clearTimeout(timeoutID);
+    configurationAccordion.click();
+    await waitToTriggerOpenAnimation();
+  }
+
+  const labelElement = this.shapeView.selectors.label ||
+    Array.from(this.shapeView.selectors.root.children).find(el => el.classList.contains('labels'));
+  labelElement.style.outline = '1px dashed blue';
```

Comment:
This seems to error when doubleclicking a message flow.

Reply:
Ah, I get this error:
```
focusOnDoubleClick.js Uncaught (in promise) TypeError: Cannot read property 'style' of undefined
```

It seems message flows don't have labels, so I'll return if `labelElement` is `undefined`.